.travis.yml

```diff
@@ -1,3 +1,5 @@
+dist: precise
```

Comment:
Yup, travis setup fails if we don't force the previous ubuntu version. See #184.